client/clientRoutes.ts

```diff
@@ -0,0 +1,88 @@
+import { generatePath } from 'react-router'
+
+import { CountryIso } from '@meta/area'
+import { AssessmentName } from '@meta/assessment'
+
+export enum AssessmentHomeRouteNames {
+  overview = 'overview',
+  messageBoard = 'messageBoard',
+  contentCheck = 'contentCheck',
+  userManagement = 'userManagement',
+  recentActivity = 'recentActivity',
+  links = 'links',
+}
+
+export const ClientRoutes = {
```

Comment:
This looks nice, the only thing I would nest the routes so that the hierarchy is clear. What do you think ?